Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -344,6 +344,137 @@ - (void)testRunWithPassingTestsSet {
     XCTAssert(exitCode == BPExitStatusTestsAllPassed);
 }
 
+- (void)testReuseSimulator {
+    //[BPUtils quietMode:NO];
```

Comment:
how about making `[BPUtils quietModel:YES]` as default in tests? (move to `+(void)setUp{}`)